### PR TITLE
first comment

### PR DESCRIPTION
**Pull request by seanderson**
seems ok

### CODE REVIEW

**Comment by seanderson** [comment]
Leaving comments is a pain.  Do I have to make a new pull request and comment across branches?

File: w1.html [hunk 1]
```diff
@@ -13,6 +13,10 @@
   <a href = "view-source:http://bardcollege.github.io/w1.html">Code</a>
 
     <code>Language</code>?
+<br>
+<br>
+<br>
+https://developer.mozilla.org/en-US/Learn/HTML/Introduction_to_HTML/Getting_started
 
-      <a href = "w2.html">--></a>
+      <a href = "w2.html">Next Page</a>
```
Comment (seanderson) [inline w1.html]: I like making them change the --> to something less confusing.

Comment (seanderson) [inline w1.html]: comment on this line

**Comment by keithohara** [comment]
I like the --> because of the ambiguity that is involved --> is also used for comments.

**Comment by seanderson** [comment]
Some changed to instructions/material on pages w2-w4

**Pull request closed without merging**